exercises/exercise-2/README.md

```diff
@@ -0,0 +1,82 @@
+# Exercise 2: Autogenerating a client SDK and consuming the mocked server
+
+In this exercise, you will connect a UI to the mocked server so that you can develop the client/UI that will consume the API without having to implement the server first. To do so, you'll generate a Client SDK directly from the API Specification, without manually coding the API calls.
```

Comment:
Could you avoid saying `mocked server` and replace it with something else like `mocked backend`. This way you leave the mocked server (a.k.a. server stub) for exercise 3. What do you think?